GrandCentralBoard/Widgets/Harvest/View/AreaBarChartComponentViewModel+FromDailyBillingStats.swift

```diff
@@ -48,15 +48,43 @@ extension AreaBarChartComponentViewModel {
         return CGFloat(count) / CGFloat(totalCount)
     }
 
+    private static var minHours: Double {
+        return 3.0
+    }
+
+    private static var maxHours: Double {
+        return 8.0
+    }
+
     private static func heightForGroup(group: BillingStatsGroup, groups: [BillingStatsGroup]) -> CGFloat {
-        let maxHours = 8.0
-        let hours = min(group.averageHours, maxHours)
+        let hours = max(minHours, min(maxHours, group.averageHours))
 
         return CGFloat(hours / maxHours)
     }
 
+    private static func doubleText(value: Double) -> String {
```

Comment:
`extension Double` and with meaningful method name.

`elegantString` or `stringWithoutPointZero`?